example/helloWorld/src/helloworld.cpp

```diff
@@ -0,0 +1,177 @@
+/**
+ * \file
+ * Copyright 2014-2015 Benjamin Worpitz
```

Comment:
Right, please use the copyright line.
If the author flag should be used, this should be done for the whole project.